ledger_validation_result.js

```diff
@@ -0,0 +1,45 @@
+const {AssetProof} = require('./asset_proof');
+
+/**
+ * To handle LedgerValidationResponse
```

Comment:
Since it's simple POJO class, we can remove the comment.

Reply:
Google JavaScript Style Guide requires all classes to have JSDoc.
https://google.github.io/styleguide/jsguide.html#jsdoc-class-comments
I think I will just use `@export` tag for that.

Reply:
Hmm, I checked the JSDoc, there's no `@export` tag.
I have no idea why Google JavaScript Style Guide is using it.
I will use `@public` instead.